KsApi/models/Project.Country.swift

```diff
@@ -36,6 +36,19 @@ extension Project {
     public static let US = Country(countryCode: "US", currencyCode: "USD", currencySymbol: "$",  maxPledge: 10_000, minPledge: 1, trailingCode: true)
     // swiftlint:enable line_length
     // swiftlint:enable comma
+
+    public static let all: [Country] = [
+      .AU, .AT, .BE, .CA, .CH, .DE, .DK, .ES, .FR, .GB, .HK, .IE, .IT, .LU, .MX, .NL, .NO, .NZ, .SE, .SG, .US
+    ]
+  }
+}
+
+extension Project.Country {
+  public init?(currencyCode: String) {
```

Comment:
Quick lil initializer to return a valid project country by currency code. Used in the reward view model logic.